Themes/default/languages/ManageSettings.english.php

```diff
@@ -56,6 +56,10 @@
 $txt['invalid_cookie_domain'] = 'The domain introduced seems to be invalid, please check it and save again.';
 $txt['secureCookies'] = 'Force cookies to be secure';
 $txt['httponlyCookies'] = 'Force cookies to be made accessible only through the HTTP protocol';
+$txt['samesiteCookies'] = 'Force cookies to be send to only first parties';
+$txt['samesiteNone'] = 'None';
+$txt['samesiteLax'] = 'Lax';
+$txt['samesiteStrict'] = 'Strict';
```

Comment:
The word just Lax,Strict,None are not very descriptive, but I don't think there is a better way to explain them without getting wordy or getting something wrong.  I'm mostly wanting to pick on Lax.  I'm assuming that it means "Relaxed".   Ignore this if nobody can think of some better wording.  The documentation describes it, but it is also confusing to new users.

Reply:
This are the offical states,
to use other words would lead people more in confusion.